doc/classes/Sky.xml

```diff
@@ -12,24 +14,35 @@
 	</methods>
 	<members>
 		<member name="radiance_size" type="int" setter="set_radiance_size" getter="get_radiance_size" enum="Sky.RadianceSize">
+			The Sky's radiance map size.
+			The higher the radiance map size, the more detailed the lighting from the Sky will be.
+			See RADIANCE_SIZE_* constants for values. Default size is RADIANCE_SIZE_512.
 		</member>
 	</members>
 	<constants>
 		<constant name="RADIANCE_SIZE_32" value="0" enum="RadianceSize">
+			Radiance texture size will be 32x32 pixels large.
```

Comment:
In general, we're trying to avoid future tense, so try changing "will be" to "is".  Also, "large" doesn't fit, nor would "small" - it's just a size.